mne/io/brainvision/brainvision.py

```diff
@@ -836,3 +836,17 @@ def read_raw_brainvision(vhdr_fname, montage=None,
     return RawBrainVision(vhdr_fname=vhdr_fname, montage=montage, eog=eog,
                           misc=misc, scale=scale, preload=preload,
                           stim_channel=stim_channel, verbose=verbose)
+
+
+def _bv_parser(description, add_for_r=1000):
+    """Parse BrainVision event codes (like `Stimulus/S 11`) to ints."""
+    if description.startswith("Stimulus") or description.startswith("Response"):
+        try:
+            split_on = "S" if description.startswith("Stimulus") else "R"
+            code = int(description.split(split_on)[-1].lstrip())
+            if split_on == "R":
+                code += add_for_r
+            return code
+        except ValueError:  # probably not coercable to int
+            pass
```

Comment:
Any chance you can simplify this with a regexp?

Reply:
How about this?

Reply:
If you're just looking for single characters, doesn't this also match a lot of event names we don't want to catch?

Reply:
Only if they end with an int. But ok, I should probably amend it so that it then additionally checks if the description starts with "resp" hm?

Reply:
Depends on how specific you want to be. I thought that these events are always called "Stimulus" and "Response", so why not catch only exact matches?

Reply:
I think they are ... Ok, I can check for exact matches.